recipe/meta.yaml

```
@@ -73,7 +72,7 @@ requirements:
     - {{ mpi }}  # [mpi != 'nompi']
     - numpy
     - setuptools
-    - tempest-remap 2.1.6  # [tempest == 'tempest']
+    - tempest-remap 2.2  # [tempest == 'tempest']
```

```suggestion
    - tempest-remap 2.2.0  # [tempest == 'tempest']
```
I think you want to include the patch version unless you want this to work for all `2.2.*`.

---

Alternatively, this should be:
```suggestion
    - tempest-remap >=2.2.0,<2.3.0  # [tempest == 'tempest']
```

---

Got it. Will push a fix.